artiq/frontend/aqctl_corelog.py

```diff
@@ -101,6 +113,8 @@ def main():
                     loop.run_until_complete(get_logs_task)
                 except asyncio.CancelledError:
                     pass
+        except:
+            logger.error("Lost connection to the core device due to: ", exc_info = True)
```

Comment:
Just say "Terminating due to exception". There can be exceptions raised for reasons other than the core device connection being lost - in fact one can even be raised (and this code executed) before the core device has been contacted at all. So your message is misleading.
Remove the spaces around =